### ISSUE TITLE
Is there any way to change autoSelectField label

### ISSUE THREAD

**Issue by syam5492009** [react-querybuilder/react-querybuilder#300]
First of all I would like to thank you for providing us a great  library. 
Is there any possibility to change the autoselectfield  label ------ from some custom text. (If I choose autoSelectField false)

**Comment by jakeboone02** [comment]
Not currently, unfortunately, but I'll put a version out shortly that will add that option. Perhaps even later today.

Thanks for the suggestion. I'm mad that I didn't think of that when I added the `autoSelectField` prop!

**Comment by jakeboone02** [comment]
@syam5492009, I just released [v4.2.5](https://github.com/react-querybuilder/react-querybuilder/releases/tag/v4.2.5) which adds a couple of new options to the `translations` prop. To change the label for the placeholder field, set the `fields.placeholderLabel` property in the `translations` prop, like this:

```tsx
import { QueryBuilder } from 'react-querybuilder';
import { fields } from './fields';

const App = () => {
  return (
    <QueryBuilder
      fields={fields}
      autoSelectField={false}
      translations={{ fields: { placeholderLabel: 'Please select a field' } }}
    />
  );
}
```

Documentation for the `autoselectfield` prop has also been updated -- see [here](https://react-querybuilder.js.org/docs/api/querybuilder#autoselectfield).

**Comment by syam5492009** [comment]
Thanks for the update.
